tf-modules/cumulus-test-cleanup/reap-kinesis.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
Given recent retro comments, I have to raise the question of unit tests -- particularly around filterOld, but the rest as well.   At a minimum  we should probably wrap runReaper in a test or two as it's the exported function.

Reply:
So we're not publishing this outside of committing it to the repo and it's documented that it's for internal use. I would find that equivalent to testing the helper functions in example or in the integration-tests package which seems like overkill.

Reply:
Except....we've started writing tests around things like lock-stack.js post retro.   I take your point on that, and probably personally agree in this instance.